paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -1170,7 +1186,6 @@ void CPUQuantizePass::ApplyImpl(ir::Graph* graph) const {
   QuantizeImmutable(graph, "reshape2", "X");
   QuantizeImmutable(graph, "transpose2", "X");
   QuantizeImmutable(graph, "slice", "Input");
-  QuantizeImmutable(graph, "shape", "Input");
```

Comment:
If shape is to be never quantized, you could remove it from all other places where it was added here
https://github.com/PaddlePaddle/Paddle/pull/44124/files